src/Data/Time/PreciseDuration.purs

```diff
@@ -27,6 +27,19 @@ instance showPreciseDuration :: Show PreciseDuration where
   show (Days d) = "(Days " <> show d <> ")"
   show (Weeks d) = "(Weeks " <> show d <> ")"
 
+toString :: PreciseDuration -> String
+toString =
+  let s = show <<< toNumber
```

Comment:
Instead of using `show`, I think the right thing to do here is to use [`toString` from `Data.BigInt`](https://pursuit.purescript.org/packages/purescript-bigints/3.4.0/docs/Data.BigInt#v:toString)

Reply:
This is a moot point given that `BigInt.toString` exists, but FWIW `BigInt.toNumber` loses precision, which is why I wrote this:

https://github.com/awakesecurity/purescript-precise-datetime/blob/baf523fefb80b544a1335371693ae4bc8cc8778d/src/Data/PreciseDateTime.purs#L148-L153

Reply:
👍 I missed that